CASMINST-7104 : passing image and config for rebuild (#5953)
Cray-HPE/docs-csm: operations/iuf/workflows/management_rollout.md

- [`management-nodes-rollout` with CSM upgrade](#21-management-nodes-rollout-with-csm-upgrade)
- [`management-nodes-rollout` without CSM upgrade](#22-management-nodes-rollout-without-csm-upgrade)

### **Note for CSM V1.7.0**

> Starting with CSM V1.7.0, administrators can use an IMS image and configuration from CFS built outside of IUF to perform the `management-nodes-rollout` stage. The image and configuration can be explicitly passed in the iuf-cli command. 

Reviewer: Remove trailing whitespace

> Starting with CSM V1.7.0, administrators can use any IMS image and configuration from CFS to perform the `management-nodes-rollout` for the same management role and subrole. The image and configuration can be explicitly passed in the iuf-cli command. 

Reviewer 2: ```suggestion
Starting with CSM V1.7.0, administrators can use an IMS image and configuration from CFS built outside of IUF to perform the `management-nodes-rollout` stage. The image and configuration can be explicitly passed in the iuf-cli command. 
```

> For example, to rebuild a storage node, the command would look like this:

Reviewer 2: ```suggestion
For example, to upgrade a storage node using an image and CFS configuration created outside of the `prepare-images` stage, the command could look like this:
```
